yammbs/_store.py

```diff
@@ -542,24 +542,52 @@ def from_qcarchive_dataset(
 
         store = cls(database_name)
 
-        for qm_molecule in tqdm(dataset.qm_molecules, desc="Storing molecules"):
-            molecule = Molecule.from_mapped_smiles(qm_molecule.mapped_smiles, allow_undefined_stereo=True)
-            molecule.add_conformer(Quantity(qm_molecule.coordinates, "angstrom"))
```

Comment:
Just confirm my understanding - with your changes, this object never sees a conformer and instead it's sent directly to the database?

Reply:
Slick!